src/pages/Onboard/AcceptInvitationPage.vue

```diff
@@ -0,0 +1,392 @@
+      return `We can't find your membership invitation. Have you already
+              accepted?`
+    },
+    noSlash() {
+      return this.$vuetify.breakpoint.xl
+        ? 15
+        : this.$vuetify.breakpoint.lg
+        ? 10
+        : 5
+    },
+    slashClass() {
+      return {
+        slash: this.$vuetify.breakpoint.smAndUp,
+        'slash-horizontal': this.$vuetify.breakpoint.smAndDown,
+        'slash-full': this.$vuetify.breakpoint.smAndDown,
+        'slash-origin-center': this.$vuetify.breakpoint.smAndUp
+      }
+    },
+    slash1Class() {
+      return {
+        ...this.slashClass,
+        paused: !this.start
+      }
+    },
+    slash2Class() {
+      return {
+        slash: this.$vuetify.breakpoint.smAndUp,
+        'slash-origin-center': this.$vuetify.breakpoint.smAndUp,
+        paused: !this.start
+      }
+    },
+    slash3Class() {
+      return {
+        ...this.slashClass,
+        paused: false
+      }
+    },
+    slash4Class() {
+      return {
+        ...this.slashResourcesClass,
+        'slash-origin-left': this.$vuetify.breakpoint.smAndUp,
+        paused: true
+      }
+    },
+    slash5Class() {
+      return {
+        ...this.slashResourcesClass,
+        'slash-origin-right': this.$vuetify.breakpoint.smAndUp,
+        paused: true
+      }
+    }
+  },
+  mounted() {
+    setTimeout(() => {
+      this.start = true
+    }, 250)
+  },
+  methods: {
+    ...mapActions('tenant', ['setCurrentTenant']),
+    async accept() {
+      this.loading = true
+      const tenant = this.membershipInvitation.tenant
+      const invitationId = this.membershipInvitation.id
+      const accepted = await this.acceptInvitation(invitationId)
+      if (accepted) {
+        sessionStorage.removeItem('invitationId')
+        await this.setCurrentTenant(tenant.slug)
+        this.$router.push({
+          name: 'dashboard',
+          params: { tenant: tenant.slug }
+        })
+      }
+      this.loading = false
+    },
+    async decline() {
+      this.deleting = true
+      const invitationId = this.membershipInvitation.id
+      const declined = await this.declineInvitation(invitationId)
+      if (declined) {
+        this.toDashboard()
+      }
+      this.deleting = false
+    },
+    toDashboard() {
+      sessionStorage.removeItem('invitationId')
+      this.$router.push({ name: 'dashboard' })
+    },
+    async declineInvitation(id) {
+      try {
+        const { data } = await this.$apollo.mutate({
+          mutation: require('@/graphql/Tenant/delete-membership-invitation.gql'),
+          variables: {
+            membershipInvitationId: id
+          }
+        })
+        if (data?.delete_membership_invitation?.success) {
+          return true
+        }
+      } catch (e) {
+        this.mutationErrorMessage = e
+          .toString()
+          .split(':')
+          .pop()
+        this.loading = false
+        this.error = true
+      }
```

Comment:
I think this method is doing 2 different things - 1 it's returning the success of the mutation, the other it's setting an error but returning nothing. I think it'd make more sense to either 1) move the error logic into the `decline` method, keeping this method idempotent, or 2) moving the `if (declined)` block into the success statement here. I could also see a scenario where we don't need the first method at all, since it's sort of a wrapper for `declineInvitation`.